userdocs/diagnostics/sendable-closure-captures.md

```diff
@@ -0,0 +1,118 @@
+# Captures in a `@Sendable` closure
+
+`@Sendable` closures can be called multiple times concurrently, so any captured values must also be safe to access concurrently. The compiler will prevent mutable captures in a `@Sendable` closure under complete concurrency checking.
+
+For example:
+
+```swift
+func callConcurrently(
+  _ closure: @escaping @Sendable () -> Void
+) { ... }
+
+func capture() {
+  var result = 0
+  result += 1
+  
+  callConcurrently {
+    print(result)
+  }
+}
+```
+
+Building with complete concurrency checking will diagnose the capture of `result` in a `@Sendable` closure:
+
+```
+|   callConcurrently {
+|     print(result)
+|           `- error: reference to captured var 'result' in concurrently-executing code
+|   }
+| }
+```
+
+Because closure is `@Sendable`, the implementation of `callConcurrently` is allowed to call `closure` multiple times concurrently, e.g. using a task group:
```

Comment:
```suggestion
Because the closure is marked `@Sendable`, the compiler allows the implementation of `callConcurrently` to call `closure` multiple times concurrently, for example using a task group:
```